reference/5.1/Microsoft.PowerShell.Utility/Out-File.md

```diff
@@ -359,14 +359,16 @@ You can pipe any object to `Out-File`.
 
 ## NOTES
 
-If you send an unformatted object to an `Out` cmdlet, the cmdlet sends it to a formatting cmdlet
-before rendering it.
+Input objects are automatically formatted as there would be in the terminal, but you can use a
```

Comment:
Sounds great, except for a typo: "as there would be" -> "as they would be".

Reply:
Good catch. I need to open my eye a little more!